universidad-de-leon-harvard.csl

```diff
@@ -0,0 +1,305 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Universidad de León (España) - Harvard</title>
+    <title-short>ULE (España) - Harvard</title-short>
+    <id>http://www.zotero.org/styles/universidad-de-leon-harvard</id>
+    <link href="http://www.zotero.org/styles/universidad-de-leon-harvard" rel="self"/>
+    <link href="http://www.zotero.org/styles/harvard-university-of-greenwich" rel="template"/>
+    <link href="https://biblioteca.ucm.es/cps/recursos-para-tfg-tfm-citas-y-referencias-bibliograficas" rel="documentation"/>
```

Comment:
@buvmc, did you enter this documentation link? It points to a website from the Universidad Complutense Madrid, not the Universidad de León. Does the Universidad de León have its own online style guide?

Reply:
Hi @rmzelle. Yes, I'd entered the documentation link from the Universidad Complutense from Madrid because I understand that is needed a link to a documentation manual and we still don't have any like that in the Universidad de León

Reply:
We do have a strong preference for the actual documentation link -- that's the only way we are able to tell what's right when someone reports an error in the style, for example. If the style guide is not online, you can also just attach it to a post here (note that you can't do that by email; you have to upload it on the github site).

Reply:
I have a little guide about the style, but it has been planned in order to work with the Mendeley software. If that is not a problem, here it is: http://buleria.unileon.es/handle/10612/5930